primitives/core/src/ed25519.rs

```diff
@@ -433,7 +433,7 @@ impl TraitPair for Pair {
 
 	/// Make a new key pair from secret seed material.
 	///
-	/// You should never need to use this; generate(), generate_with_phrasee
+	/// You should never need to use this; generate(), generate_with_phrase
```

Comment:
This comment could be more clear in general (although maybe not directly related to this PR). Is this saying I should use `generate_with_phrase` instead?